WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -96,6 +99,8 @@ extension FeatureFlag {
             return "Jetpack disconnect"
         case .debugMenu:
             return "Debug menu"
+        case .readerCSS:
+            return "Always fetches a fresh Reader CSS"
```

Comment:
I am a bit confused by what this meant. Perhaps this is a better description?

```suggestion
            return "Ignore Reader Detail CSS Cache"
```

Reply:
Done in 8fa1b5f1a95504ff15d32c530bbe6174e0319606

I just removed the word "Detail" to keep it more concise for everyone (iOS devs or not).